src/Redux/Categories/categoriesTypes.js

```diff
@@ -0,0 +1,3 @@
+const SHOW_STATUS = 'SHOW_STATUS';
+
+export default SHOW_STATUS;
```

Comment:
- [ ] Kindly refer to the comments above and fix this. A lot of explanation of this practice has been outlined clearly above.